### DISCUSSION TITLE
Action: Export to Excel for opportunities records

### DISCUSSION THREAD

**Discussion by GournieGR** [YetiForceCompany/YetiForceCRM#16754] [How to]
Hi,

I have a problem when i export to excel the records who exist in opportunities module.
Specifically, the columns who return the excel trim the column compaign source to specific characters :
![image](https://user-images.githubusercontent.com/60574307/208685915-b7363a0c-9af4-479c-bcaf-1f35f06d9acd.png)

Is there a way to avoid that and to return all characters ?

thank you

**Comment by rskrzypczak** [comment]
You are probably using an old version of the application. Version 6.4 does not have this problem. If you want it, check it out on the [demo](https://gitstable.yetiforce.com/) version. 
So - update your system :)

**Reply by GournieGR** [reply]
Is there a way to achieve that without update my system ?
